_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,68 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector as an Integrator. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
```

Comment:
@britsta how to resolve these TODOs? It seems like these are the only thing blocking this PR.

Reply:
leave todo open, will be fixed in https://github.com/nmshd/documentation/pull/170

Reply:
ok, great.

Reply:
so then the only open thing is the duplicated information (tables)